patches/@types+mocha+9.0.0.dev.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/node_modules/@types/mocha/index.d.ts b/node_modules/@types/mocha/index.d.ts
+index 2d989c1..5d01724 100755
+--- a/node_modules/@types/mocha/index.d.ts
++++ b/node_modules/@types/mocha/index.d.ts
+@@ -2621,7 +2621,7 @@ declare var suiteTeardown: Mocha.HookFunction;
+  *
+  * @see https://mochajs.org/api/global.html#beforeEach
+  */
+-declare var beforeEach: Mocha.HookFunction;
++// declare var beforeEach: Mocha.HookFunction;
```

Comment:
is there any way to say these only affect mocha files and shouldnt be considered for jest tests?